app/config.py

```diff
@@ -0,0 +1,14 @@
+from functools import lru_cache
+
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class Settings(BaseSettings):
+    secret_key: str
+
+    model_config = SettingsConfigDict(env_file="../../.env")
```

Comment:
you don't need .env when running the app in a container; set ENVs for the app in docker-compose.yml